client/src/pages/Profile/Profile.jsx

```diff
@@ -251,6 +251,11 @@ const ProfilePageLeaderPermissionDashboardLinks = () => {
         authScopes={['faq:delete', 'faq:edit']}
         label="FAQ Admin Panel"
       />
+      <ProfilePageDashboardLink
+        link="/timeline-admin"
+        authScopes={['timeline:edit']}
```

Comment:
```suggestion
        authScopes={['timeline:edit', 'timeline:create', 'timeline:delete']}
```

there are options for create, edit, and delete for timeline... on the leedur permission page `/permission-request` (?)
![image](https://user-images.githubusercontent.com/86009011/185830251-ef70c2a0-f25d-42e2-95e5-461d6085c97d.png)

so it might be good to include all of them (as in suggestion)? or alternatively we can change the timeline scopes in the file (client > src > pages ? ScopeRequest > functions.jsx > getTotalScopes) to only edit?

Reply:
The desired behaviour here would be that `timeline:edit` allows you to edit the contents of a timeline event, `timeline:create` allows you to create a timeline event, and `timeline:delete` allows you to delete a timeline event. Good catch @natapokie !

Reply:
Got it, made the change!